docutils-stubs/nodes.pyi

```diff
@@ -11,6 +11,7 @@ from typing import Any, Callable, Dict, Iterable, Iterator, List, Optional, over
 
 __docformat__: str
 
+T_co = TypeVar('T', bound=nodes.Node, covariant=True)
```

Comment:
* By convention, the name of the first argument of TypeVar should be a name of variable on the left-hand side.
* TypeVar with name `T_co` looks like covariant type variable having upper bound `object`. A name indicating that its upper bound is `nodes.Node` is more preferable.

Examples can be taken from the implementation of `typing` module::

   CT_co = TypeVar('CT_co', covariant=True, bound=type)

https://github.com/python/cpython/blob/f2a9d5c8378cd7eca90b3b197e2cc0989da55014/Lib/typing.py#L1181